content/08-cardano-sidechains/03-example-evm-sidechain/05-sidechain-token.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: SC_Token and faucet
+metaTitle: SC_Token and faucet
+---
+
+## What is SC_Token?
+
+**SC_Token<sup>1</sup>** is the native token of the Cardano EVM sidechain Testnet. SC_Token can be moved between the main chain (Cardano) and the example EVM sidechain, and SC_Token is required to pay for gas.
+
+**Test SC_Token**, or 'Sidechain Testnet Token' SC_Token, is the native token of the example EVM sidechain testnet. Test SC_Token carries no real-world value.
+
```

Comment:
Confusing. SC_Token is required to pay for gas on the sidechain, tADA is still required to pay for tx fees on mc. So it should be moved to the second paragraph, where you describe the sidechain evm token.

Reply:
I updated this page to be not only about SC_Token but also about Test Ada